### ISSUE TITLE
when nltk failed to download nltk-data, it outputs error info to stdout?

### ISSUE THREAD

**Issue by fseasy** [nltk/nltk#2029]
my nltk version is 3.2.5. and I put a code of 

```python
nltk.download('punkt') # if already exist, it will return True.
```

to let the user automatically download it.

but I found that, it output the error info to stdout, I thought it may a problem.

the error info like:

```
[nltk_data] Error loading punkt: <urlopen error [SSL:
[nltk_data]     CERTIFICATE_VERIFY_FAILED] certificate verify failed
[nltk_data]     (_ssl.c:590)>
```

**Comment by alvations** [comment]
c.f. https://github.com/nltk/nltk/issues/1961#issuecomment-367207040

Are you using Python3.6 on Mac OS? If so, it looks like similar to this issue: https://bugs.python.org/issue28150 

See 

- https://stackoverflow.com/questions/41348621/ssl-error-downloading-nltk-data
- https://stackoverflow.com/a/41692664/610569


TL;DR
====

```
/Applications/Python 3.6/Install Certificates.command
```

**Comment by fseasy** [comment]
@alvations  thx, but I'm using Python2.7 on CentOS 4.3.
May be I hadn't expressed my intention，I am not for the error, but for that the error info output device: not to stderr, but to stdout. I am using `stdin -> procedure -> stdout`  to run my code, so it messed up my streaming procedure.

**Comment by stevenbird** [comment]
@memeda, that's interesting. We always thought of the NLTK corpus downloader as being used interactively. Even successful use generates output on stdout.

One fix is a flag to specify non-interactive use.

**Comment by fseasy** [comment]
@stevenbird Yeah, I thought so. but I want to reduce the users' preparatory operations to run my code, so I put it to the module's initialization. 
A flag is great. may be just output to stderr will ok? The interactive terminal could display the stderr info right.

**Comment by alvations** [comment]
@memeda  we do have the `quiet` argument for `nltk.download()` function. But I think having controls to know where the error messages are piped to is useful.

@stevenbird I think expecting error messages to go to `stderr` could be the default. And `stdout` 
or otherwise as defined by users. How about something like #2031?

**Comment by fseasy** [comment]
@alvations great

**Comment by stevenbird** [comment]
Adopted, thanks @alvations and @memeda.

**Comment by Dilshad737** [comment]
[nltk_data] Error loading stopwords: <urlopen error [SSL:
[nltk_data]     CERTIFICATE_VERIFY_FAILED] certificate verify failed
[nltk_data]     (_ssl.c:646)>
